Makefile

```diff
@@ -21,6 +21,9 @@ help: # Display help
 			printf "\033[36m%-30s\033[0m %s\n", $$1, $$NF \
 		}' $(MAKEFILE_LIST) | sort
 
+.PHONY: setup
+setup: setup-minikube setup-tekton-chains tekton-generate-keys setup-kyverno example-buildpacks ## Sets up the SSF project into minikube
```

Comment:
This does a little bit more than just "setup". It actually triggers a pipeline run.

Reply:
Ah yup, that's true. 🤔 Any idea/suggestion to rename it to something else?

Reply:
Quickstart?

Reply:
Haha, straightforward! I like that 👍